tests/model_forms/tests.py

```diff
@@ -3496,6 +3496,46 @@ class InheritedForm(NewForm):
                 type(NewForm.base_fields[name].widget),
             )
 
+    def test_custom_callback_in_meta(self):
+        call_count = 0
+
+        def callback(db_field, **kwargs):
+            nonlocal call_count
+            call_count += 1
+            return db_field.formfield(**kwargs)
```

Comment:
We should add a real-life callback, not mocking that `callback()` is called.

Reply:
Fixed, thanks!